ext/dep_gecode/dep_selector_to_gecode.cpp

```diff
@@ -633,6 +649,8 @@ VersionProblem * VersionProblem::InnerSolve(VersionProblem * problem, int &iterc
                     delete best_solution;
```

Comment:
Minor (not your code): delete doesn't really need a NULL check.. also since you're in here it would be nice to fix the open brace placement on the while loop :)